CHANGES.md

```diff
@@ -20,8 +21,13 @@ Notable fixes and improvements to existing languages:
 
 [Bram de Haan]: https://github.com/atelierbram
 [Kenneth Fuglsang]: https://github.com/kfuglsang
+<<<<<<< HEAD
```

Comment:
I was looking for more overlooked artifacts. I don’t think there’s any more by me, but I found some others:
```
$ grep -r --binary-files=without-match '<<<< '
node_modules/gear-lib/node_modules/less/test/less/property-name-interp.less.orig:<<<<<<< HEAD
node_modules/gear-lib/node_modules/less/lib/less/tree/rule.js.orig:<<<<<<< HEAD
node_modules/gear-lib/node_modules/less/lib/less/tree/rule.js.orig:<<<<<<< HEAD
node_modules/gear-lib/node_modules/less/lib/less/parser/parser.js.orig:<<<<<<< HEAD:lib/less/parser/parser.js
```
Are those intentional?